lib/fs/modules/LinuxMount.rb

```diff
@@ -309,23 +309,27 @@ def expandLinks(p)
     #
     components.each do |c|
       ncp = File.join(cp, c)
-      #
-      # Each file system know how to check for,
-      # and read its own links.
-      #
-      fs, lp = getFsPathBase(ncp)
-      if fs.fileSymLink?(lp)
-        sl = getSymLink(fs, lp)
-        if sl[0, 1] == '/'
-          cp = sl
-        else
-          cp = File.join(cp, sl)
-        end
+      cp = follow_all_symlinks(ncp)
+    end
+    (cp)
+  end
+
+  def follow_all_symlinks(link_ptr)
+    #
+    # Each filesystem knows how to check for,
+    # and read its own links.
+    #
+    no_more_links = nil
```

Comment:
A nit, but can you set this to `true`?

Reply:
If I set it to true then the loop will never get executed.  And the name of the variable will be incorrect.  Or are you suggesting I change the name of the variable and change the "until" to a "while"?

Reply:
Sorry, I meant `false` not `true`. I just meant to say use a boolean instead of `nil`.

Reply:
Right.  Done.  :-)